.github/workflows/lint.yml

```diff
@@ -21,5 +21,9 @@ jobs:
           node-version: '14'
       - name: Install markdownlint
         run: npm install -g markdownlint-cli
+      - name: Install markdownlint
+        run: npm install -g markdown-link-check
       - name: markdownlint
         run: just lint-docs
+      - name: markdown-link-check
+        run: just lint-docs
```

Comment:
did you mean `just linkcheck-docs`?

Reply:
whoops -- I swear I had this right locally 🤔

Reply:
that's better I think?